dropwizard-auth/src/main/java/io/dropwizard/auth/AuthFilter.java

```diff
@@ -138,6 +138,7 @@ protected boolean authenticate(ContainerRequestContext requestContext, C credent
                 return false;
             }
 
+            boolean secure = requestContext.getSecurityContext().isSecure();
```

Comment:
Also, do we need to check that a `SecurityContext` has been set before calling this to avoid a NPE?